scripts/tell/tell.php

```diff
@@ -42,7 +42,7 @@ function gtell($args, \Irc\Client $bot, \knivey\cmdr\Request $req)
         }
         $max = $config['tell_max_inbox'] ?? 10;
         $net = $bot->getOption('NETWORK', 'UnknownNet');
-        if ((countMsgs($req->args['nick'], $net, true) >= $max) || !strcmp($req->args['nick'], 'terps')) {
+        if ((countMsgs($req->args['nick'], $net, true) >= $max) && !strcmp($req->args['nick'], 'terps')) {
```

Comment:
can we use strcasecmp() here which despite the stupid name is the case-insensitive version.